src/services/query.ts

```diff
@@ -109,8 +107,12 @@ export class QueryTermsService {
   }
 }
 
-const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+const alphabeticallyByLabel = (a: Term, b: Term) => {
```

Comment:
Or: `alphabeticallyByLabels` (in plural), since it sorts on both a prefLabel and altLabel?

Reply:
Goed idee!